10.9-libcxx/stable/main/finkinfo/libs/pythonmods/pygments-py.info

```diff
@@ -28,12 +35,12 @@ PostInstScript: <<
 <<
 PreRmScript: <<
  if [ $1 != "upgrade" ]; then
-  update-alternatives --remove pygmentize %p/bin/pygmentize%type_raw[python]
+  update-alternatives --remove pygmentize %p/bin/pygmentize-py%type_pkg[python]
```

Comment:
Don't know if this is an issue or not, but does this leave any residues behind if you have the old version that installs as `%p/bin/pygmentize3.5` and now you're installing the new version with the file `%p/bin/pygmentize-py35` ? The `PreRmScript` from the previous version won't be run (only happens when not upgrading). I don't know how u-a deals with new sets of files taking over an existing namespace.

Reply:
You're right; had been wondering about this briefly, but I was worried that a manpage like `pygmentize3.8.1` might not be correctly recognised.
I am trying to check for old installed versions of the binaries now, but apparently a test like
```bash
 if [ -f %p/bin/pygmentize%type_raw[python] ]; then
  update-alternatives --remove-all pygmentize
 fi
```
does not respond – is this a limitation of the restricted interpreter for `PostInstScript`, or do you know if it has a different syntax for tests?

Reply:
Don't think there are restrictions on P{re,ost}InstScript. The contents of the .info field are just added to a shell script header (see %p/var/lib/dpkg/info for many examples). So that's just normal `#!/bin/sh` call. What happens if you just `ls -l %p/bin/pygmentize*` to debug?
Also, fixing the symlinks probably belongs in PreInstScript for the updated `pygmentize-py.info`

Reply:
> Also, fixing the symlinks probably belongs in PreInstScript for the updated `pygmentize-py.info`

Uh, of course - it was looking for the old file after it had already been replaced from the new package!
In PreInstScript it works as it should (I hope at least, as `update-alternatives --remove` seems to work somewhat unpredictably regarding the changes to `%p/etc/alternatives/pygmentize`...).